ui/components/app/confirmations/alerts/alert-modal/alert-modal.tsx

```diff
@@ -95,58 +126,74 @@ function AlertHeader({ selectedAlert }: { selectedAlert: Alert }) {
           marginTop={3}
           marginBottom={4}
         >
-          {selectedAlert.reason ?? t('alert')}
+          {customAlertTitle ?? selectedAlert.reason ?? t('alert')}
         </Text>
       </Box>
     </>
   );
 }
 
-function AlertDetails({ selectedAlert }: { selectedAlert: Alert }) {
+function AlertDetails({
+  selectedAlert,
+  customAlertDetails,
```

Comment:
rather than custom<XYZ> why not turn AlertModal into a presentational component so that it is not dependent on Alert? or... why does ConfirmAlertModal not utilize the selectedAlert system? Is it possible only multiple-alert-modal utilizes the "selectedAlert" logic?

e.g. something like, instead of the current MultipleAlertModal

```
    <AlertModal
      ownerId={ownerId}
      onAcknowledgeClick={handleAcknowledgeClick}
      alertKey={selectedAlert.key}
      onClose={onClose}
      headerStartAccessory={
        <PageNavigation
          alerts={alerts}
          onBackButtonClick={handleBackButtonClick}
          onNextButtonClick={handleNextButtonClick}
          selectedIndex={selectedIndex}
        />
      }
    />
  );
  ```
  
  
```
    <AlertModal
      ownerId={ownerId}
      onAcknowledgeClick={handleAcknowledgeClick}
      title={selectedAlert.reason}
      details={selectedAlert.alertDetails}
      message={selectedAlert.message}
      severity={selectedAlert.severity}
      onClose={onClose}
      headerStartAccessory={
        <PageNavigation
          alerts={alerts}
          onBackButtonClick={handleBackButtonClick}
          onNextButtonClick={handleNextButtonClick}
          selectedIndex={selectedIndex}
        />
      }
    />
  
  ```